### PR TITLE
[Bluetooth][TCSACR-238] Remove the deprecated APIs in the guide

### PR DESCRIPTION
**Pull request by Wootak**
Remove the deprecated APIs in the guide
 - ACR: TCSACR-238

### CODE REVIEW

File: docs/application/dotnet/guides/connectivity/bluetooth.md [hunk 1]
```diff
@@ -1078,22 +1078,14 @@ To discover nearby LE devices, perform an LE scan operation:
             }
         }
 
-        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList();
+        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetGetServiceDataList(BluetoothLePacketType.BluetoothLeAdvertisingPacket);
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/connectivity/bluetooth.md]: Would you please check this GetGetServiceDataList?

Comment (Wootak) [reply docs/application/dotnet/guides/connectivity/bluetooth.md]: Done. Thank you!

File: docs/application/dotnet/guides/connectivity/bluetooth.md [hunk 2]
```diff
@@ -1078,22 +1078,14 @@ To discover nearby LE devices, perform an LE scan operation:
             }
         }
 
-        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList();
+        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList(BluetoothLePacketType.BluetoothLeAdvertisingPacket);
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/connectivity/bluetooth.md]: Is it ok to use GetServiceDataList? I can see this method is also in the deprecated list in http://suprem.sec.samsung.net/jira/browse/TCSACR-238. If I misunderstood something please let me know.

Comment (Wootak) [reply docs/application/dotnet/guides/connectivity/bluetooth.md]: Yes. GetServiceDataList(void) was duplicated as you said. And, GetServiceDataList(BluetoothLePacketType packetType) is added on below ACR.
http://suprem.sec.samsung.net/jira/browse/TCSACR-237
Thank you.

**Comment by Wootak** [comment]
Thank you very much!
Please confirm the ACR also.
http://suprem.sec.samsung.net/jira/browse/TCSACR-238

**Comment by mijin85cho** [comment]
http://suprem.sec.samsung.net/jira/browse/TCSACR-238 is Doc confirmed.

**Pull request merged**